compiler/codegen/mli.cpp

```diff
@@ -0,0 +1,851 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mli.h"
```

Comment:
We typically insert a newline to delineate the sections I mentioned earlier (see the examples I mentioned)

Reply:
I am not sure why that is not showing up, perhaps I did forget to push? All these things are changed locally for me...

Reply:
By changed locally, do you mean Github shows they are different?  I tried refreshing the page multiple times so that would be obnoxious on its part

Reply:
By changed locally, I meant on my local copy of the repo. However I scrolled back and did see the "endif" change pushed 3 commits ago, so I'm not sure.